src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts

```diff
@@ -153,17 +194,22 @@ export class SettingsTreeIndicatorsLabel {
 					},
 					markdownNotSupportedFallback: contentFallback
 				};
-				let hover: ICustomHover | undefined = undefined;
 				const options: IUpdatableHoverOptions = {
-					linkHandler: (scope: string) => {
+					linkHandler: (url: string) => {
+						const [scope, language] = decodeURIComponent(url).split(':');
 						onDidClickOverrideElement.fire({
-							targetKey: element.setting.key,
-							scope
+							settingKey: element.setting.key,
+							scope: scope as ScopeString,
+							language
 						});
-						hover!.dispose();
+						this.hover!.hide();
 					}
 				};
-				hover = setupCustomHover(this.hoverDelegate, this.scopeOverridesElement, content, options);
+				if (this.currentHoverMarkdownString !== contentMarkdownString) {
+					this.hover?.dispose();
```

Comment:
Does the hover need to be tracked somewhere else to be sure it's disposed?

Reply:
Most likely. I'll see if I can just add it to elementDisposables for now.

Reply:
Current plan: implement `IDisposable` and add it to another `DisposableStore` outside of the class.